.chloggen/main.yaml

```
@@ -0,0 +1,27 @@
+# Use this changelog template to create an entry for release notes.
+
+# One of 'breaking', 'deprecation', 'new_component', 'enhancement', 'bug_fix'
+change_type: enhancement
+
+# The name of the component, or a single word describing the area of concern, (e.g. filelogreceiver)
+component: zipkinexporter
+
+# A brief description of the change.  Surround your text with quotes ("") if it needs to start with a backtick (`).
+note: "The Zipkin exporter currently ignores the scope attributes on OTLP payloads which leads to missing information. This PR adds scope attributes as tags to avoid any info getting corrupted/missing."
```

Make it a tad shorter?
```suggestion
note: "Include instrumentation scope attributes"
```